autocertkit/network_tests.py

```diff
@@ -739,8 +739,9 @@ def _verify_ethtool_offloads(self, session, device):
                 # If it is not possible due to the hardware ristriction, then ACK should
                 # ignore this failure and keep running tests.
                 if '[fixed]' in hw_offloads[k]:
```

Comment:
Couple of questions:
* Should we differentiate between offloads we rely on, and everything else? (e.g. whitelist the ones that can be fixed).
* If they are fixed, do we need to verify whether they are fixed on or not? (e.g. we shouldn't allow LRO to be fixed on).

Other than that, this looks fine.

Thanks.